cdbsearch.py

```diff
@@ -498,7 +498,14 @@ async def cdbsearch(epd, depthLimit, concurrency, evalDecay, cursedWins=False):
             pvlen = len(pv) - 1
             if pv[-1] == "checkmate":
                 if await chessdb.pv_has_proven_mate(board.epd(), pv):
-                    pv[-1] = "CHECKMATE"
+                    pv[-1] = "CHECKMATE" + (
+                        ""
+                        if pvlen == 0
+                        else f" (#{(pvlen+1)//2})"
```

Comment:
would it make sense to show this at the score line instead?

Reply:
I had exactly the same thought. But then I thought it sits maybe nicer next to CHECKMATE, and if people use scripts or some such to data mine cdbsearch output they can always be sure it `eval` a numerical value.